charts/camunda-platform/README.md

```diff
@@ -70,6 +70,7 @@ Check out the default [values.yaml](values.yaml) file, which contains the same c
 | | `identity.auth.tasklist.redirectUrl` |  Defines the redirect URL, which is used by Keycloak to access Tasklist. Should be public accessible, the default value works if port-forward to Tasklist is created to 8082. Can be overwritten if, an Ingress is in use and an external IP is available. | `"http://localhost:8082"` |
 | | `identity.auth.optimize.existingSecret` |  Can be used to reference an existing secret. If not set, a random secret is generated. The existing secret should contain an `optimize-secret` field, which will be used as secret for the Identity-Optimize communication. | ` ` |
 | | `identity.auth.optimize.redirectUrl` |  Defines the redirect URL, which is used by Keycloak to access Optimize. Should be public accessible, the default value works if port-forward to Optimize is created to 8083. Can be overwritten if, an Ingress is in use and an external IP is available. | `"http://localhost:8083"` |
+| | `identity.keycloak.fullnameOverride` |  Overrides the Keycloak service name, the default value is {{ .Release.Name}}-keycloak | `"{{ .Release.Name}}-keycloak"` |
```

Comment:
:x: This is not really true, right? It just configures the other services to use that name instead. The keycloak configuration is still necessary by the user. 

Maybe we also need to rename `fullnameOverride` but not 100% sure, do you have an opinion? For example just name, fullname or serviceName.

Reply:
I think `fullnameOverride` will show the user that this should be the same as `fullnameOverride` in keycloak config. But we can change the explanation for sure.

Reply:
Yeah you are right, will change it